test_data/pil/binary.pil

```diff
@@ -1,9 +1,9 @@
 // A compiled version of std/machines/large_field/binary.asm
 
 namespace main(128);
-    col witness a, b, c;
+    col witness binary_op, a, b, c;
     // Dummy connection constraint
-    [0, a, b, c] is main_binary::latch * main_binary::sel[0] $ [main_binary::operation_id, main_binary::A, main_binary::B, main_binary::C];
+    [binary_op, a, b, c] is main_binary::latch * main_binary::sel[0] $ [main_binary::operation_id, main_binary::A, main_binary::B, main_binary::C];
```

Comment:
Just to make it clear that the operation ID does not need to be known at compile time.